sonar-ws/src/main/java/org/sonarqube/ws/client/project/ProjectsService.java

```diff
@@ -106,4 +106,12 @@ public SearchWsResponse search(SearchWsRequest request) {
       .setParam(PAGE_SIZE, request.getPageSize());
     return call(get, SearchWsResponse.parser());
   }
+
+  public void changeVisibility(ChangeVisibilityRequest request) {
+    PostRequest post = new PostRequest(path("change_visibility"))
+        .setParam(PARAM_PROJECT, request.getProject())
+        .setParam("visibility", request.getVisibility());
```

Comment:
I would prefer, that `getVisibility()` returns the enum constant, and we extract the string value here by calling something like `getVisibility().getApiValue()`.